pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/custom/CpcSketchTest.java

```diff
@@ -181,7 +181,7 @@ public File createAvroFile()
       for (int i = 0; i < getCountStarResult(); i++) {
         // create avro record
         GenericData.Record record = new GenericData.Record(avroSchema);
-        record.put(ID, RandomUtils.nextInt(10));
+        record.put(ID, RandomUtils.nextInt(0, 10));
```

Comment:
Seems `RandomUtils` is already deprecated. Can you replace it with a regular `Random`?

Reply:
I did notice the deprecation warning. I plan to use [commons-rng](https://commons.apache.org/proper/commons-rng/) as it's much more performant than `java.util.Random` in the follow-up PR. Thoughts?

Reply:
Sounds good!